### ISSUE TITLE
Accessibility: Cannot set alt="" attribute

### ISSUE THREAD

**Issue by Abdull** [netresearch/t3x-rte_ckeditor_image#167]
(Using [`netresearch/rte-ckeditor-image:10.2.1`](https://github.com/netresearch/t3x-rte_ckeditor_image/tree/v10.2.1) (which is named [`10.1.0` according to its `ext_emconf.php`](https://github.com/netresearch/t3x-rte_ckeditor_image/blob/v10.2.1/ext_emconf.php#L10)))

When using `rte-ckeditor-image`, it is not possible to give an `<img>` element an empty `alt` attribute, e.g. `<img src="decoration.jpg" alt="">`.

For accessibility reasons, it is important to be able to [set `alt=""` on an image in order to indicate that this image is not content-relevant](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt) (also see [W3C WAI's *An `alt` Decision Tree*](https://www.w3.org/WAI/tutorials/images/decision-tree/)).

[Wrong usage of `alt`-Attributes lead to a website to fail WCAG / BITV accessibility conformance](https://webtest.bitv-test.de/index.php?a=di&iid=239).

Without `rte-ckeditor-image`, *TYPO3 10*'s default CKEditor 4 inline image dialog (via https://ckeditor.com/docs/ckeditor4/latest/examples/image.html ?) seems to create empty `alt` attributes when no value is provided. This is a sane default.

**Comment by olagwin** [comment]
I could see this behaviour also in the version for TYPO3 11

**Comment by Gitsko** [comment]
Hello, I have just checked it again, in the current version 11 and 12 an empty alt tag is generated if no value has been entered in the dialog.

`<img alt="" data-alt-override="true" data-htmlarea-file-table="sys_file" data-htmlarea-file-uid="166" data-title-override="true" src="/fileadmin/_processed_/6/d/csm_picard_s1_brd_front_stlbk-pc-games6_4959380117.jpg" style="" width="299" height="168">`


We are currently unable to provide support for version 10.
